cmd/nodecmd/wiz.go

```diff
@@ -228,9 +226,14 @@ func wiz(cmd *cobra.Command, args []string) error {
 
 	if !clusterAlreadyExists {
 		globalNetworkFlags.UseDevnet = true
-		if len(useCustomAvalanchegoVersion) == 0 && !useLatestAvalanchegoReleaseVersion && !useLatestAvalanchegoPreReleaseVersion {
-			useAvalanchegoVersionFromSubnet = subnetName
-		}
+		//if len(useCustomAvalanchegoVersion) == 0 && !useLatestAvalanchegoReleaseVersion && !useLatestAvalanchegoPreReleaseVersion {
+		//	useAvalanchegoVersionFromSubnet = subnetName
+		//}
+		// Hardcode version for devnet to avoid resolution issues
+		useCustomAvalanchegoVersion = "v1.13.5"
```

Comment:
hardcoding due to error in getting appropriate avalnachego version if we just call `node devnet wiz` without first calling `blockchain create` command